eZ/Publish/Core/FieldType/RichText/Resources/stylesheets/docbook/xhtml5/edit/core.xsl

```diff
@@ -436,39 +436,59 @@
     </xsl:element>
   </xsl:template>
 
-  <xsl:template match="docbook:ezembed | docbook:ezembedinline">
-    <xsl:element name="{local-name()}" namespace="{$outputNamespace}">
-      <xsl:if test="@xlink:href">
-        <xsl:attribute name="href">
-          <xsl:value-of select="@xlink:href"/>
-        </xsl:attribute>
-      </xsl:if>
-      <xsl:if test="@xml:id">
-        <xsl:attribute name="id">
-          <xsl:value-of select="@xml:id"/>
-        </xsl:attribute>
-      </xsl:if>
-      <xsl:if test="@view">
-        <xsl:attribute name="data-ezview">
-          <xsl:value-of select="@view"/>
-        </xsl:attribute>
-      </xsl:if>
-      <xsl:if test="@ezxhtml:class">
-        <xsl:attribute name="class">
-          <xsl:value-of select="@ezxhtml:class"/>
-        </xsl:attribute>
-      </xsl:if>
-      <xsl:if test="@ezxhtml:align">
-        <xsl:attribute name="data-ezalign">
-          <xsl:value-of select="@ezxhtml:align"/>
-        </xsl:attribute>
-      </xsl:if>
+  <xsl:template match="docbook:ezembed">
+    <xsl:element name="div" namespace="{$outputNamespace}">
+      <xsl:attribute name="data-ezelement">
+        <xsl:value-of select="'ezembed'"/>
```

Comment:
Simplified in https://github.com/ezsystems/ezpublish-kernel/commit/0f2c9f69c90e5fc44feefca3196b03fd3a781115